### PR TITLE
Add retry logic to the command that sets content_origin in ephemeral.

### PR DESCRIPTION
**Pull request by dkliban**
fixes: #94

### CODE REVIEW

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -119,7 +119,17 @@ objects:
                         number: 24816
       EOF
 
-      kubectl patch pulp cs-pulp --type='json' -p='[{"op": "replace", "path": "/spec/pulp_settings/content_origin", "value": "'https://"${HOST}"'"}]'
+      # Sometimes the content_origin is not getting set. We need to check the value and try again if needed.
+      EPHEMERAL_CONTENT_ORIGIN=https://${HOST}
+      while true; do
```
Comment (decko) [inline deploy/clowdapp.yaml:124]: Could this get into some sort of infinity loop?

Comment (dkliban) [reply deploy/clowdapp.yaml:124]: yes

**Comment by jlsherrill** [comment]
oh @decko this has a conflict now it seems?

**Comment by jlsherrill** [comment]
@dkliban this has a conflict now it seems

**Comment by dkliban** [comment]
closing because i don't think there is actually a problem here. there were other issues going on when this problem was reported.

**Pull request closed without merging**